### DISCUSSION TITLE
Does quadrant  support FLAT index (100% recall rate, without losses)?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [qdrant/qdrant#2981] [General]
Does quadrant  support FLAT index (100% recall rate, without losses)?

**Comment by timvisee** [comment]
No, Qdrant does not support a flat index at this time. We provide an HNSW index though. It should be general purpose enough to fit 99% of the use cases.

You can do exact searches though (with `?exact=true`) to bypass the HNSW index and do a full scan to ensure 100% accuracy.
